src/js/flights/flight.js

```diff
@@ -0,0 +1,59 @@
+import getPlace from './places';
+import flightData from './flightData';
+const moment = require('moment');
+
+const getFlight = (placeID, departureDate) => {
+  return fetch(
+    `https://skyscanner-skyscanner-flight-search-v1.p.rapidapi.com/apiservices/browsedates/v1.0/PL/PLN/en-GB/WRO-sky/${placeID}/${departureDate}?inboundpartialdate=anytime`,
+    {
+      method: 'GET',
+      headers: {
+        'x-rapidapi-host':
+          'skyscanner-skyscanner-flight-search-v1.p.rapidapi.com',
+        'x-rapidapi-key': 'f76b9a08c6msh4342c2bb7ef2877p1c960ajsn0922b19cbacf',
+      },
+    }
+  )
+    .then(response => {
+      return response.json();
+    })
+    .catch(err => {
+      return err;
+    });
+};
+
+const getFlightData = async city => {
+  let departureDate = moment()
+    .add(1, 'day')
+    .format()
+    .substr(0, 10);
+  let minPrice;
+  let placeID;
+  let placeName;
+  let country;
+  let carrierID;
+  let carrierName;
+
+  let data = await getPlace(city);
+  placeID = data.Places[0].PlaceId;
```

Comment:
nie musisz tego deklarowac wczesniej, mozesz tutaj i od razu warto zrobic z tego consta

Reply:
w zasadzie to było tylko dla mojej wygody podczas pisania kodu - już poprawione